packages/synapse-react-client/src/utils/functions/SqlFunctions.ts

```diff
@@ -38,26 +38,20 @@ export const getIgnoredQueryFilterSearchParamKey = (
  * @returns
  */
 export const getAdditionalFilters = (
-  localStorageKey: string,
+  sessionStorageKey?: string,
   searchParams?: Record<string, string>,
   operator: SQLOperator = ColumnSingleValueFilterOperator.LIKE,
 ): QueryFilter[] | undefined => {
-  const localStorageQueryFiltersString = localStorage.getItem(
-    QUERY_FILTERS_LOCAL_STORAGE_KEY(localStorageKey),
-  )
+  const sessionStorageQueryFiltersString = sessionStorageKey
+    ? sessionStorage.getItem(
+        QUERY_FILTERS_SESSION_STORAGE_KEY(sessionStorageKey),
+      )
+    : undefined
   let additionalFilters: QueryFilter[] = []
-  if (localStorageQueryFiltersString) {
+  if (sessionStorageQueryFiltersString) {
     additionalFilters = JSON.parse(
-      localStorageQueryFiltersString,
+      sessionStorageQueryFiltersString,
     ) as QueryFilter[]
-    // delay clearing out the value so re-rendering during React Strict mode does not destroy the filter
```

Comment:
This is the final significant change.  Since it's in session storage, leave it!  If the same window is refreshed (or the user browses somewhere and returns to this page) the stored filter will be re-applied.  Until the user removes the filter, or overwrites it with another selection.